sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/models/ListContainersIncludeType.java

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.storage.blob.implementation.models;
```

Comment:
Looks like this is double generated? Shouldn't there only be the `ListBlobContainersIncludeType` in the public API and this isn't needed?

Reply:
Removed.